lib/missing_env_error.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
```

Comment:
nit: could we standardize on snake case filenames as part of this?

Reply:
I'll standardize on kebab case as usual.

Reply:
done.